tests/xbot/common/subsystems/ConnectionMonitorTest.java

```diff
@@ -23,18 +22,35 @@ public void setUp() {
     }
 
```

Comment:
These are good tests that you've added, but what I meant was to have at least one test where setLastPacketReceivedTimestamp is called more than once.

Your current tests do a good job of evaluating if your class responds well to some immediate impulse, but they don't test how your class behaves if its primary method is called several times in a row.